uui/components/overlays/Alert.module.scss

```diff
@@ -10,13 +10,23 @@
 }
 
 .alert-wrapper {
-    padding: 12px 18px 12px 12px;
     display: flex;
+    align-items: center;
```

Comment:
Why do we need align-items: center here? We don't have it before.

Reply:
![image](https://github.com/epam/UUI/assets/94478026/fa585010-d22b-468f-8a7c-bcda73c8af79)
![image](https://github.com/epam/UUI/assets/94478026/b714276e-3b12-4f9c-8548-84dd00392acb)

after removing align-items the text is not at center. if you align item can cause any problem then it can be replace by some padding